Jenkinsfile

```diff
@@ -17,6 +17,22 @@ pipeline {
 
             parallel {
 
+                stage('Security Check: Main') {
```

Comment:
The reason we split up between the two is that we want to impose more strict security requirements on the framework than on the tests.